### PR TITLE
minor tweek

### PR DESCRIPTION
**Pull request by joinimran**
As the user has suggested I have added a description link for the product life cycle. 

Problem: https://github.com/MicrosoftDocs/OfficeDocs-Enterprise/issues/535

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/upgrade-from-sharepoint-2010.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ When your SharePoint Server 2010, and SharePoint Foundation 2010 software reache
...
   
 > [!NOTE]
-> A software lifecycle typically lasts for 10 years from the date of the product's initial release. You can search for [Microsoft solution providers](https://go.microsoft.com/fwlink/?linkid=841249) who can help with upgrade to the next version of your software, or with Office 365 migration (or both). Be certain you're aware of end of support dates on critical underlying technologies as well, particularly of the version of SQL Server you're using with SharePoint. 
+> A software lifecycle typically lasts for 10 years from the date of the product's initial release. You can search for [Microsoft solution providers](https://go.microsoft.com/fwlink/?linkid=841249) who can help with upgrade to the next version of your software, or with Office 365 migration (or both). Be certain you're aware of end of support dates on critical underlying technologies as well, particularly of the version of SQL Server you're using with SharePoint. See [Fixed Lifecycle Policy](https://support.microsoft.com/help/14085) to understand product lifecycle in detail.
```
Comment (JohanFreelancer9) [inline Enterprise/upgrade-from-sharepoint-2010.md]:
```suggestion
> A software lifecycle typically lasts for ten years from the date of the product's initial release. You can search for [Microsoft solution providers](https://go.microsoft.com/fwlink/?linkid=841249) who can help with upgrading to the next version of your software, or with Office 365 migration (or both). Be certain you're aware of end of support dates on critical underlying technologies as well, particularly of the version of SQL Server you're using with SharePoint. See [Fixed Lifecycle Policy](https://support.microsoft.com/help/14085) to understand product lifecycle in detail.
```

**Comment by mypil** [comment]
@joinimran - please merge the commit on the copy/edit review made by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by joinimran** [comment]
Ready to go live.

**Comment by mypil** [comment]
@MSFTTracyP - PR has been copy edited and is ready for final review. This is based on issue #535

Thank you.

cc: @kenwith

**Pull request merged**